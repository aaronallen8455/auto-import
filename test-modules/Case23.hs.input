module Case23 where

main :: T :+: T
main = pure ()
