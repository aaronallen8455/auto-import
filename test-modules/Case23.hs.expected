module Case23 where

import Namespace (type (:+:), type T)

main :: T :+: T
main = pure ()
